tests/test_dcos_e2e/backends/aws/test_aws.py

```diff
@@ -32,6 +32,12 @@ def test_admin_location(self) -> None:
         """
         assert AWS().admin_location == '0.0.0.0/0'
 
+    def test_aws_instance_type(self) -> None:
```

Comment:
This test will not pass.

In particular, there is no `aws_instance_type` attribute set in the `AWS.__init__`.

What you need to do is, in `AWS.__init__`, do `self.aws_instance_type = aws_instance_type` and document `aws_instance_type` in the `Attributes` section of the docstring.